web/bfex/components/data_pipeline/tasks/keywords.py

```diff
@@ -13,6 +13,9 @@ class UpdateKeywordsFromScrape(Task):
     """
     def __init__(self):
         self.task_name = "Update Keywords From Scrape"
+        self.approaches = [RakeApproach(), GenericApproach()]
+        self.ids = ConfigFile().data['keygen_ids']
+        self.filtered_algs = [alg for alg in self.approaches if alg.get_id() in self.ids]
```

Comment:
You arent actually using the KeywordGenerator here. We should be using that, register each of the configured approaches with the KeyGenerator so we can reuse it.

Lane is using the KeyGenerator in #37 maybe take a look at that and work with him to get this functionality working.